app/controllers/account_home_controller.rb

```diff
@@ -0,0 +1,40 @@
+class AccountHomeController < ApplicationController
+  include GovukPersonalisation::AccountConcern
+  before_action -> { set_slimmer_headers(template: "gem_layout_account", remove_search: true, show_accounts: "signed-in") }
+
+  def show
+    @is_account = true
+    @user = GdsApi.account_api.get_user(govuk_account_session: account_session_header).to_h
+  rescue GdsApi::HTTPUnauthorized
+    logout!
+    redirect_with_ga GdsApi.account_api.get_sign_in_url(redirect_path: account_home_path).to_h["auth_uri"]
+  end
+
+  helper_method :show_confirmation_reminder?
+
+  def show_confirmation_reminder?
```

Comment:
can we make move this to private method?
The tests still pass if we do

That way it keeps the controller public method section to RESTful method names.
It is slightly odd having a private method that is then called as a helper method from the view, but I think that makes more sense?

Or alternatively, could we make these take the `@user` in as a parameter and move them out of the controller into a Helper module alltogether?

(I think they will still have access to @user if they are included in as a helper... but I always find helpers as external modules calling instance methods even stranger... who knows where you might include it into!)